compiler/rustc_target/src/spec/loongarch64_unknown_linux_gnu.rs

```diff
@@ -0,0 +1,18 @@
+use crate::spec::{CodeModel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "loongarch64-unknown-linux-gnu".into(),
+        pointer_width: 64,
+        data_layout: "e-m:e-p:64:64-i64:64-i128:128-n64-S128".into(),
+        arch: "loongarch64".into(),
+        options: TargetOptions {
+            code_model: Some(CodeModel::Medium),
```

Comment:
To me it will make generated code larger and slower.  I know there are things (Mozilla `libxul.so` for e.g.) requiring this, but should we hard code it globally?

Reply:
@xen0n how do you think about this?  I know medium code model is needed for at least these Mozilla monsters, but I really dislike to slow everything down just for them.

Reply:
It's already gone in the latest revision.